packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteView/useDeleteView.ts

```diff
@@ -40,10 +36,7 @@ export const useDeleteView = ({
 
   const onActionStart = () => {
     if (!current) return;
-    handleProcess({
-      config: getInput(),
-      prefix: key,
-    });
+    handleProcess({ config: getInput() });
```

Comment:
prefix is no longer needed because we have the full key, right?

Reply:
Right, `key` is directly pulled from the list output items